packages/language-server/src/completion/completions.ts

```diff
@@ -149,21 +149,27 @@ function getSuggestionForBlockAttribute(
   return suggestions
 }
 
-
 export function getSuggestionForNativeTypes(
   foundBlock: Block,
   wordsBeforePosition: string[],
   document: TextDocument,
   binPath: string,
-  lines: string[]
+  lines: string[],
 ): CompletionList | undefined {
   const activeFeatureFlag = declaredNativeTypes(document, binPath)
-  if (foundBlock.type !== 'model' || !activeFeatureFlag || wordsBeforePosition.length < 2) {
+  if (
+    foundBlock.type !== 'model' ||
+    !activeFeatureFlag ||
+    wordsBeforePosition.length < 2
+  ) {
     return undefined
   }
 
   const datasourceName = getFirstDatasourceName(lines)
-  if (!datasourceName || wordsBeforePosition[wordsBeforePosition.length - 1] !== `@${datasourceName}`) {
+  if (
```

Comment:
Is there an extra space here or is this just looking weird in the diff?

Reply:
Just looking weird here, I checked 👍